File: b/Java/RAM.java
public class RAM {
	// Atribut
	private int capacity;
	private int price;

	// Konstruktor
	public RAM() {
		capacity = 0;
		price = 0;
	}

	// Setter & Getter
	public void setCapacity(int capacity) {
		this.capacity = capacity;
	}
	public void setPrice(int price) {
		this.price = price;
	}

	public int getCapacity() {
		return capacity;
	}
	public int getPrice() {
		return price;
	}

	// Method 
	public void printRAM() {
		System.out.println("- RAM -");
		System.out.println("Capacity    : " + capacity + " GB");
	}
}

File: b/PHP/RAM.php
<?php

class RAM {
	// Atribut
	private $capacity;
	private $price;

	// Konstruktor
	function __construct() {
		$capacity = 0;
		$price = 0;
	}

	// Setter & Getter
	function setCapacity($capacity) {
		$this->capacity = $capacity;
	}
	function setPrice($price) {
		$this->price = $price;
	}

	function getCapacity() {
		return $this->capacity;
	}
	function getPrice() {
		return $this->price;
	}

	// Method
	function displayRAM() {
		echo "<br>RAM";
		echo 
		"
		<table border='1'>
			<tr>
				<td><b>Capacity(GB)</b></td>
				<td>".$this->capacity."</td>
			</tr>
			<tr>
				<td><b>Price(Rp)</b></td>
				<td>".$this->price."</td>
			</tr>
		</table> 
		";
	}
}

?>